optimum/habana/transformers/models/mpt/modeling_mpt.py

```diff
@@ -26,23 +26,34 @@
 
 from ...modeling_attn_mask_utils import _gaudi_prepare_4d_causal_attention_mask
 
-
 logger = logging.get_logger(__name__)
 
+try:
+    from habana_frameworks.torch.hpex.kernels import FusedSDPA
+except ImportError:
+    print("Not using HPU fused scaled dot-product attention kernel.")
+    FusedSDPA = None
+
+import habana_frameworks.torch.core as htcore
 
 def gaudi_mpt_attention_forward(
     self,
     hidden_states: torch.Tensor,
     position_bias: torch.Tensor,
     past_key_value: Optional[Tuple[torch.Tensor]] = None,
     attention_mask: Optional[torch.Tensor] = None,
+    output_attentions: bool = False,
```

Comment:
output_attentions is never passed from mpt_block_forward(). do you need to add it?